mevislab.github.io/content/tutorials/basicmechanisms.md

```diff
@@ -157,35 +153,35 @@ The connection is drawn as a thin grey arrow between both modules with the arrow
 
 ![Parameter Connection StartSlice](/images/tutorials/basicmechanics/BM_14.png "Parameter Connection StartSlice")
 
-We now like to establish a connection from the module `SyncFloat` to the second viewer, `Viewer2D1`. In order to do that, open the automatic panel `View2D1`. Draw a connection from the label `Float2` of the panel of the module `SyncFloat` to the label *startSlice* in the automatic panel of the module `View2D1`. At last, implement a connection between the parameter fields *startSlice* of both viewers. Draw the connection from `View2D1` to `View2D`.
+We will now establish a connection from the module `SyncFloat` to the second viewer, `Viewer2D1`. In order to do that, open the automatic panel `View2D1`. Draw a connection from the label `Float2` of the panel of the module `SyncFloat` to the label *startSlice* in the automatic panel of the module `View2D1`. Lastly, implement a connection between the parameter fields *startSlice* of both viewers. Draw the connection from `View2D1` to `View2D`.
 
 ![Synchronize both directions](/images/tutorials/basicmechanics/BM_15.png "Synchronize both directions")
 
 As a result, scrolling through the slices with the mouse wheel {{< mousebutton "middle" >}} in one of the viewers synchronizes the rendered slice in the second viewer. In this case, you can inspect the differences between smoothed and unsmoothed data on every single slice.
 
 ![Your final Network](/images/tutorials/basicmechanics/BM_16.png "Your final Network")
 
-You can also use the pre-defined Module `SynchroView2D` to reach the same goal (shown in [this chapter](/tutorials/visualization/visualizationexample1/) ).
+It is also possible to use the pre-defined module 'SynchroView2D' to accomplish a similar result.(`SynchroView2D`s usage is futherly described in [this chapter](/tutorials/visualization/visualizationexample1/) ).
```

Comment:
backticks for `SynchroView2D`

Reply:
But the modules name is already put in between backticks?